NuRadioReco/modules/LOFAR/stationPulseFinder.py

```diff
@@ -275,7 +276,8 @@ def run(self, event, detector):
             station_even_list = []
             station_odd_list = []
             for channel in station.iter_channels():
-                if channel.get_id() == channel.get_group_id():
+                # if channel.get_id() == channel.get_group_id(): #tested it, does not work
+                if channel.get_id() == int(channel.get_group_id()[1:]): #this works
```

Comment:
I am sorry, but this cannot be correct. In `readLOFARData.py` we use this line: 
```python
channel_group: int = detector.get_channel_group_id(station_id, channel_id)
```
So `channel.get_group_id()` should return an integer and not a string. Do you perhaps test your code using old NUR files which were processed with the old reader code?

Reply:
Ah, yes that is possible. Ok, I will revert this again.